domains/Discussions/Tests/Feature/AnswersUpdateTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Domains\Discussions\Tests\Feature;
+
+use Domains\Accounts\Database\Factories\UserFactory;
+use Domains\Accounts\Models\User;
+use Domains\Discussions\Database\Factories\AnswerFactory;
+use Domains\Discussions\Database\Factories\QuestionFactory;
+use Domains\Discussions\Models\Answer;
+use Domains\Discussions\Models\Question;
+use Faker\Factory;
+use Faker\Generator;
+use Illuminate\Http\Response;
+use Illuminate\Support\Carbon;
+use Laravel\Lumen\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class AnswersUpdateTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    private Generator $faker;
+    private User $user;
+    private Question $question;
+    private Answer $answer;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->faker = Factory::create();
+        $this->user = UserFactory::new()->create();
+        $this->question = QuestionFactory::new(['author_id' => $this->user->id])->create();
+        $this->answer = AnswerFactory::new([
+            'author_id' => $this->user->id,
+            'question_id' => $this->question->id,
+        ])->create();
```

Comment:
Suggestion:

maybe avoid calling all these Factories by calling the AnswerFactory first
`$this->answer = AnswerFactory::new()->create()`
and then do
`$this->user = $this->answer->author`
and
`$this->question = $this->answer->question`

Reply:
You're right!